style.css

```diff
@@ -0,0 +1,457 @@
+* {
+    box-sizing: border-box;
+}
+
+html,
+body {
+    margin: 0;
+    height: 100%;
+}
+
+body {
+    font-family: 'Open Sans', Oswald;
+    background: #EFEFEF;
+    margin: 0 auto;
+    width: 100%;
+    color: #A9A9A9;
+}
+
+.container {
+    min-height: 100%;
+    margin-bottom: -75px;
```

Comment:
This is really interesting. I'd love to hear how you made this choice. It's not bad, just interesting!